File: JavaNIDE/Javaprgm/app/src/main/java/sample/Forloop8.java
package sample;
import java.util.*;
class Forloop8
{
  public static void main (String[] ar)
  {
   int i,j,n,k=1,prod=1;
   double sum=0;
System.out.print("enter range");
Scanner m=new Scanner(System.in);
 n=m.nextInt();
   for(i=1;i<=n;i++)
   {
     if(i%2!=0)
       {
     System.out.print(k+"^"+k);
     sum=sum+Math.pow(k,k);
     k++;
     }
     else
{
  for(j=1;j<=k;j++)
  {
   prod=prod*j;
  System.out.print(j);
  if(j<k)
  {
    System.out.print("*");
  }
}
sum=sum+prod;
prod=1;
     
   }
   if(i<n)
   {
     System.out.print(" + ");
   }
  }
  System.out.println("sum of seriese="+sum);
}
}

File: python programming/forloop_8.py
n=int(input("enter no of term:-"))
sum=0
k=3
for i in range(1,n+1):
 print(f'{i}*{i+1}/{k}*{k+1}',end='')
 if i<n:
  print(' + ',end='')
 sum=sum+((i)*(i+1))/((k)*(k+1))
 k=k+2
print("\nsum of seriese=",sum)
